README.md

```diff
@@ -1,655 +1,288 @@
-         txt_s = as.Date(x$text_dates$start), txt_e = as.Date(x$text_dates$end)) %>% 
-      mutate(
-          txt_in = txt_s - ifelse(is.null(x$fade_in), 9, x$fade_in), 
-          txt_out = txt_e + ifelse(is.null(x$fade_out), 9, x$fade_out)
-      )
-}) %>% bind_rows() %>% 
-    # Order the figure output based on event start date
-    arrange(start) %>% 
-    mutate(label = factor(label, levels = label, ordered = TRUE))
-
-library(ggplot2)
-ggplot(dates_of_events, aes(y = 1, yend = 1)) +
-  geom_segment(aes(x = start, xend = end), size = 3) +
-  geom_segment(aes(y = 0.5, yend = 0.5, x = txt_s, xend = txt_e), size = 2, color = "blue") +
-  geom_segment(aes(y = 0.5, yend = 0.5, x = txt_in, xend = txt_out), size = 1, color = "red", linetype = "dotted") +
-  ylim(0, 2) +
-  geom_text(aes(x = start, y = 1.5, label = label), hjust = 0) +
-  facet_grid(label ~ .) + 
-  theme(axis.text=element_blank(), axis.ticks=element_blank(),
-        strip.background = element_blank(), strip.text = element_blank(),
-        axis.title = element_blank(), panel.grid = element_blank(),
-        panel.spacing = unit(0, "lines"))
+Next, actually download the data by running the following code which kicks off the lengthy data processing steps. Note that this will take multiple hours. I usually plan on kicking this off in the morning and then getting back to it right at the end of the day, or even the next day. It also prints a lot to the console.
 
-# Build a frame for the middle of each event
+```
+source('helper_fxns_pipeline.R')
+rebuild_gage_data()
+```
+### Create a new blank animation to use for event callout inspiration
 
-dates_to_build <- lapply(lapply(yaml::read_yaml("callouts_cfg.yml"), '[[', "text_dates"), function(x) {
-  endDate <- as.Date(x$end)
-  startDate <- as.Date(x$start)
-  halfwayDate <- startDate + (endDate - startDate)/2
-  return(halfwayDate)
-}) %>% unlist() %>% as.Date(origin = "1970-01-01") %>% format("%Y%m%d")
+Unless you have already know what the callouts will be, you should generate a blank version of the animation to use and share with collaborators in order to generate event callouts. If you already have callouts, still do the configuration steps here but don't bother rebuilding the animation frames until after you've followed the instructions further down to incorporate callouts into the animation.
 
-# Don't forget the font: 
-sysfonts::font_add_google('Abel','abel')
-scipiper::scmake(sprintf('6_visualize/tmp/frame_%s_00.png', dates_to_build), '6_timestep_gif_tasks.yml', force=TRUE)
+#### Configuration step: clear `6_visualize/tmp`
 
-```
+In the end, the animation is created by stitching together a bunch of individual PNG frames. The code is currently setup to just use any file within the folder `6_visualize/tmp`. Thus, we need to clear (or rename) that folder to prevent old frames from appearing in our new visualization. If this folder doesn't exist for you, please create it and leave it empty for now.
```

Comment:
If you are coming back to this for another quarter after you built the previous one, you will have `6_visualize/tmp` and it will be populated. I've updated the text to make that clear